### PR TITLE
Add setup / teardown info to graph_data endpoint

### PR DESCRIPTION
**Pull request by dstandish**
Needed for UI magic from @bbovenzi

### CODE REVIEW

File: airflow/utils/task_group.py [hunk 1]
```diff
@@ -653,15 +653,21 @@ def task_group_to_dict(task_item_or_group):
...
             "value": {
-                "label": task_item_or_group.label,
-                "labelStyle": f"fill:{task_item_or_group.ui_fgcolor};",
-                "style": f"fill:{task_item_or_group.ui_color};",
+                "label": task.label,
+                "labelStyle": f"fill:{task.ui_fgcolor};",
+                "style": f"fill:{task.ui_color};",
                 "rx": 5,
                 "ry": 5,
+                "setupTeardownType": setup_teardown_type,
```
Comment (dstandish) [inline airflow/utils/task_group.py]: @bbovenzi would you say i should only include this element if not None?

Comment (bbovenzi) [reply airflow/utils/task_group.py]: Yeah exactly, that's consistent with the type file below.

Comment (dstandish) [reply airflow/utils/task_group.py]: ok thanks, updated

**Review by bbovenzi — APPROVED** [review]
lgtm. I'll do the UI part as a follow up PR

**Pull request merged**